packages/calcite-components/src/components/stepper/stepper.scss

```diff
@@ -1,3 +1,14 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-stepper-action-background-color: defines the background color of an action sub-component inside the component.
+ * @prop --calcite-stepper-action-indicator-color: defines the indicator color of an action sub-component inside the component.
```

Comment:
A user can't set the indicator color of this internally rendered action. I think the same applies for text-color.